repo/repo.project.ts

```diff
@@ -0,0 +1,21 @@
+import { PrismaClient, Project } from '@prisma/client';
+import {GetServerSideProps} from "next";
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+    const prisma = new PrismaClient();
+    const projects = await prisma.project.findMany({ take: 30 });
+    return {
+        props: {projects: projects}
+    }
+}
+export const projects = findProjects().then((projects)=>{
+    return projects;
+});
+
+async function findProjects() : Promise<Project[]> {
+    const prisma = new PrismaClient();
+    return await prisma.project.findMany({take: 30});
+}
+//
+
+
```

Comment:
hi @sachdevlaksh just curious to know why did you put this outside of projects page?

Reply:
I am not sure if we can do this on projects page, I thought we are doing db call, so it usually happen on repo layer, so created separate repo module/dir for that